app/main.py

```diff
@@ -2,0 +1,18 @@
+def copy_file(command: str) -> None:
+    list_from_command = command.split(" ")
+    command_name, file_name, copy_file_name = list_from_command
+    is_name_valid = file_name != copy_file_name
+    is_command_valid = command_name == "cp"
+    is_all_params = len(list_from_command) == 3
```

Comment:
Ensure that the input string contains exactly 3 elements separated by spaces, and that 'cp' is the first element, as per the checklist .